src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsPartiesMappingTest.java

```diff
@@ -464,23 +500,19 @@ void getIndividualLastName(String value, String expected) {
     void getIndividualPreferredHearingChannelNullHearingTypeTest() {
         HearingOptions hearingOptions = HearingOptions.builder().build();
 
-        assertThatExceptionOfType(
-            IllegalStateException.class).isThrownBy(() ->
-            getIndividualPreferredHearingChannel(null,
+        assertThat(getIndividualPreferredHearingChannel(null,
                 HearingSubtype.builder().build(),
-                hearingOptions));
+                hearingOptions)).isNull();
```

Comment:
Sould the name for this test be updated?
@DisplayName("When hearingType not set then throw IllegalStateException")